azure/azure.go

```diff
@@ -75,8 +77,8 @@ func NewClientFromMetadata() (cloudops.Ops, error) {
 
 	if onAzure, metadata, err := onAzure(); onAzure && err == nil {
 		logrus.Info("Running on Azure IaaS VM")
-		var vmID, scalesetName, subscriptionID, resourceGroupName, managedClusterName, agentPoolName string
-		var scalesetNametemp, vmIDtemp, subscriptionIDtemp interface{}
+		var vmID, scalesetName, subscriptionID, resourceGroupName, managedClusterName, agentPoolName, userAgent string
+		var scalesetNametemp, vmIDtemp, subscriptionIDtemp, userAgenttemp interface{}
```

Comment:
```suggestion
		var scalesetNametemp, vmIDtemp, subscriptionIDtemp, userAgentTemp interface{}
```

Reply:
i went with the template of the other variable names.